### DISCUSSION TITLE
Frigate restream

### DISCUSSION THREAD

**Discussion by dmsviento** [blakeblackshear/frigate#8184] [Ask A Question]
Hi folks,

New to Frigate and very impressed so far.

I'm struggling with the function of re-stream and the use case(s) for it (HA). Is the use case for cameras that do no support more that one connection. If the camera supports more that one connection surely point the HA at the original rtsp from the camera?

Sorry if I've missed the point

**Comment by NickM-27** [comment]
Regardless of if your camera supports multiple connections, they can still stress the cameras encoder / bandwidth and general network activity. It is generally best to use restream when multiples of a stream will be used.
